content/SCALE/SCALETutorials/Storage/Pools/Datasets/CreatingSnapshots.md

```diff
@@ -0,0 +1,140 @@
+   
+   ![AddSnapshotScreen](/images/SCALE/22.12/AddSnapshotScreen.png "Add a New Snapshot")
+
+2. Select dataset or zvol from the **Dataset** dropdown list. 
+
+3. Accept the name suggested by the TrueNAS software in the **Name** field or enter any custom string to override the suggested name.
+
+4. (Optional) Select an option from  the **Naming Schema** dropdown list that the TrueNAS software populated with existing periodic snapshot task schemas.  
+   If you select an option, TrueNAS generates a name for the snapshot using that naming schema from the selected Periodic Snapshot and replicates that snapshot. 
+
+   You cannot enter a value in **Naming Schema** and in **Name** as selecting or entering a value in **Naming Schema** populates the other field. 
+
+5. (Optional) Select **Recursive** to include child datasets with the snapshot.
+
+6. Click **Save** to create the snapshot.
+  
+## Managing Snapshots
+
+The **Snapshots** screen lists all snapshots created on the system after you clear any dataset path from the search field at the top of the screen. 
+
+To manage snapshots, click anywhere on a snapshot to expand it and display the options for managing that snapshot.
+
+![SnapshotsScreenExpandedSnapshot](/images/SCALE/22.12/SnapshotsScreenExpandedSnapshot.png "Expanded Snapshot Screen")
+
+Select the checkbox to the left of each snapshot to select multiple snapshots and display the **Batch Operations** option to **Delete** the selected snapshots.
+
+Click the **Show extra columns** toggle to add more information in the table. Click **Show** to display extra columns in the table. To hide the added columns, click the toggle again and then click **Hide**. 
+
+Snapshot details includes the space used, date created, retention policy, and the amount of data the dataset can access. It also includes four options: **Hold**, **Delete**, **Clone To New Dataset**, and **Rollback**. 
+
+### Deleting a Snapshot
+
+{{< include file="/_includes/DeletingSnapshots.md" type="page" >}}
+
+![DeleteSnapshotDialog](/images/SCALE/22.12/DeleteSnapshotDialog.png "Delete Single Snapshot")
+
+**Confirm** activates the **Delete** button.
+
+#### Batch Operations - Delete
+To delete more than one snapshot in one operation, select the checkbox beside the datasets you want to delete and to display the **Batch Operations Delete** option. 
+
+![SnapshotsScreenBatachOperations](/images/SCALE/22.12/SnapshotsScreenBatachOperations.png "Batch Operations Delete Snapshot")
+
+**Batch Operations Delete** opens a window listing all selected snapshots.
+
+![DeleteMultipleSnapshotsWindow](/images/SCALE/22.12/DeleteMultipleSnapshotsWindow.png "Batch Operations Delete Snapshot Window")
+
+{{< include file="/_includes/SnapshotsBulkOperations.md" type="page" >}} 
+
+**Confirm** activates the **Delete** button. If a snaphot has the **[Hold](#snapshot-details-screen)** option selected, an error displays to prevent you from deleting that snapshot.
```

Comment:
```suggestion
**Confirm** activates the **Delete** button. If a snapshot has the **[Hold](#snapshot-details-screen)** option selected, an error displays to prevent you from deleting that snapshot.
```